src/main/java/com/softserve/itacademy/exception/handler/GenericExceptionHandler.java

```diff
@@ -19,26 +23,78 @@
 public class GenericExceptionHandler {
 
     @ExceptionHandler(NotFoundException.class)
-    public ResponseEntity<Object> handleAccessDeniedException(NotFoundException exception) {
-        ErrorRequest errorRequest = new ErrorRequest(exception.getMessage());
-        return new ResponseEntity<>(errorRequest, NOT_FOUND);
+    public ResponseEntity<BasicExceptionResponse> handleAccessDeniedException(NotFoundException exception) {
```

Comment:
handleAccessDeniedException is the same name like on 38 line

Reply:
Fixed, I guess this one was before me.